### DISCUSSION TITLE
⁉️Doubt or queries? Here you go

### DISCUSSION THREAD

**Discussion by ErrorxCode** [ErrorxCode/JxInsta#7] [Q&A]
Ask all your questions, doubts or queries without any hesitation. If any question is asked in the issues then it will be ignored and closed.

**Comment by ahmetyildizyazilim** [comment]
Does this coding want acces token for instagram.

**Reply by ErrorxCode** [reply]
First of all, you should check [this](https://www.ef.com/wwen/blog/language/how-to-improve-your-spoken-english/) or [this](https://support.google.com/translate/answer/6350658?hl=en&co=GENIE.Platform%3DAndroid).

Answering you question, no it only requires Instagram account **username and password**

**Comment by xiliangliang** [comment]
Hello, may I ask how to create a group and send messages within it

**Reply by ErrorxCode** [reply]
Currently, you cannot a create a group with this library. Please wait for next update. Any contribution is appriciated

**Comment by ahmetyildizyazilim** [comment]
dependencies {
    implementation files('libs/JxInsta-v1.0-beta.jar')
    implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '5.0.0-alpha.11' //skip if already implemented
} 
this code shows jar lineage where is the jar file I can't find the jar file

**Reply by ahmetyildizyazilim** [reply]
where can I download the jar file

**Reply by ErrorxCode** [reply]
from the release

**Comment by BlackMightyRavenDark** [comment]
What Java/JDK version is required to build/run this project?

**Reply by ErrorxCode** [reply]
17
